### PR TITLE
move content creation to its own section of the handbook

### PR DESCRIPTION
**Pull request by robmarcer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ and strongly encouraged.
 ### Sales and Marketing
 
  - [Sales](./sales)
+ - [Content](./content)
```
Comment (ZJvandeWeg) [inline README.md:27]: Can we call it `marketing`?

Comment (joepavitt) [reply README.md:27]: I would vote for "Marketing & Communication"

Comment (robmarcer) [reply README.md:27]: I'm with @joepavitt, I'll edit the PR.

Comment (robmarcer) [reply README.md]: Updated, OK now?

Comment (ZJvandeWeg) [reply README.md]: ✔️

**Pull request merged**